processing/src/main/java/io/druid/query/extraction/LookupExtractor.java

```diff
@@ -30,9 +30,10 @@
 import java.util.List;
 import java.util.Map;
 
-@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type", defaultImpl = LookupDelegator.class)
```

Comment:
We have the default set to `LookupDelegator` which is cool, but it would be even more nice if the specs could just use the lookup name and not have to build the full JSON object.  Can we verify that such works with the current setup (and if it doesn't, it should be possible to make it work by creating a static method that takes the manager and just `Object` and then does the right thing based on the class of the `Object`.